Source/ee/Vif.cpp

```diff
@@ -464,7 +488,7 @@ void CVif::ExecuteCommand(StreamType& stream, CODE nCommand)
 		Cmd_MPG(stream, nCommand);
 		break;
 	default:
-		assert(0);
```

Comment:
I think we should replace the assert with a `CLog::Warn`.

Reply:
The assert has been replaced with `m_STAT.nER1 = 1;` - this should be the right behavior for an unknown command according to docs.

Reply:
Indeed, but it's rare that games send invalid data on purpose. It's a useful debug tool to have a message that says that invalid data has been received, because it means there's a good chance something is going wrong (either in the VIF or somewhere else in the emulator).

Reply:
Fair point, I'll add it in